frontend/src/components/MonacoEditor.tsx

```diff
@@ -1,7 +1,11 @@
 import React from 'react'
 import Editor from '@monaco-editor/react'
 
-const MonacoEditor = ({ content }: any) => {
+interface Props {
```

Comment:
Could this be made more generic and usable in many different components?

Reply:
I am referring to the declaration of the Props-interface outside of the types-directory